PackageExplorer/App.xaml.cs

```diff
@@ -87,17 +88,6 @@ Task<IEnumerable<ICredentialProvider>> getProviders()
             
         }
 
-        private static void MigrateSettings()
```

Comment:
I accidentally delete it.
Should I add it back?

Reply:
Is it needed? :)

Reply:
Yes, added back.